doc/reflex_docs.rst

```diff
@@ -319,7 +319,7 @@ use ``foldDyn``
 
 Even nested state machines can be designed if your have a state with nested ``Dynamic`` value by using ``foldDynM``
 
-See `nested_dynamic.hs <https://github.com/reflex-frp/reflex-frp.org/blob/master/code-snippets/nested_dynamics.hs>`_
+See `nested_dynamic.hs <https://github.com/dfordivam/reflexfrp.org/blob/master/code-snippets/nested_dynamics.hs>`_
```

Comment:
I don't know about .rst, but github markdown (.md) supports relative links.

Reply:
This was a temporary fix. I will add a link to the examples in https://github.com/reflex-frp/reflex-examples/pull/17, once that is merged.